release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,6 +6,12 @@ uid: DCP_change_log
 
 The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 25 April 2023 - Enhancements -  CloudGateway 2.10.8 - General improvements [ID_36014] [ID_36259]
```

Comment:
As the text below describes both an enhancement and a fix, I'd suggest to change this to: 

```suggestion
#### 25 April 2023 - Enhancement/fix -  CloudGateway 2.10.8 - General improvements [ID_36014] [ID_36259]
```